src/bokeh/colors/color.py

```diff
@@ -365,6 +365,13 @@ def brightness(self) -> float:
         r, g, b = self.r, self.g, self.b
         return sqrt(0.299*r**2 + 0.587*g**2 + 0.114*b**2)/255
 
+    @property
+    def luminance(self) -> float:
+        """ Perceived luminance of a color in [0, 1] range. """
+        # https://en.wikipedia.org/wiki/Relative_luminance
+        r, g, b = self.r, self.g, self.b
+        return (0.2126*r**2.2 + 0.7152*g**2.2 + 0.0722*b**2.2) / 255**2.2
```

Comment:
```suggestion
        return sqrt(0.299*self.r**2 + 0.587*self.g**2 + 0.114*self.b**2)/255

    @property
    def luminance(self) -> float:
        """ Perceived luminance of a color in [0, 1] range. """
        # https://en.wikipedia.org/wiki/Relative_luminance
        return (0.2126*self.r**2.2 + 0.7152*self.g**2.2 + 0.0722*self.b**2.2) / 255**2.2
```
I think we don't need extra variables.

Reply:
Agree, did the change for brightness too